source/_components/minio.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: "Minio"
+description: "Component for interacting with Minio object storage."
+date: 2019-04-21 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: System Monitor
+ha_iot_class: Cloud Push
+ha_release: 0.92
+ha_qa_scale: internal
+---
+
+This component adds interaction with [Minio](https://min.io).
+Also enables to listen for bucket notifications: [watch docs](https://docs.min.io/docs/minio-client-complete-guide.html#watch)
+
+To download or upload files, folders must be added to [whitelist_external_dirs](/docs/configuration/basic/).
+
+## {% linkable_title Configuration %}
+
+To enable the Minio component in your installation, add the following to your `configuration.yaml` file:
+
+{% raw %}
+```yaml
+minio:
+  host: localhost
+  port: 9000
+  access_key: SO9KNO6YT9OGE39PQCZW
+  secret_key: EzH5iGs3qbJr1MwIMf2AUyfqzewgd2qgrk9+i+vU
+  secure: false
+  listen:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.